src/Beman/Optional26/tests/optional_constexpr.t.cpp

```diff
@@ -0,0 +1,1744 @@
+// src/Beman/Optional26/tests/optional_constexpr.t.cpp                -*-C++-*-
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <Beman/Optional26/optional.hpp>
+
+#include <functional>
+#include <ranges>
+#include <tuple>
+
+#include <gtest/gtest.h>
+
+TEST(OptionalConstexprTest, TestGTest) { ASSERT_EQ(1, 1); }
+
+namespace {
+struct empty {};
```

Comment:
Please check if you can use / extend types from `Beman/Beman.Optional26/src/Beman/Optional26/tests/test_types.hpp`.

I know this file is a copy of `optional.t.cpp`, but maybe we should do the updates for both files.

Reply:
They diverge slightly, but there are a lot of Don't Repeat Yourself violations. 
Refactor as part of this commit, or follow on?

Reply:
As you prefer.